sci-misc/boinc/boinc-7.14.2.ebuild

```diff
@@ -0,0 +1,192 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+WX_GTK_VER=3.0-gtk3
+
+inherit autotools eutils gnome2-utils linux-info systemd user wxwidgets
```

Comment:
any reason for `eutils` still? All the eutils stuff you're callign should be in `desktop.eclass`

Reply:
Nice catch, I confirmed that `eutils` isn't necessary and removed it.